scalameter-core/src/main/scala/org/scalameter/MeasureBuilder.scala

```diff
@@ -38,12 +43,12 @@ class MeasureBuilder[T, U](
   
       if (ctx(Key.exec.requireGC)) compat.Platform.collectGarbage()
   
-      val measurements = measurer.measure[T, U](
+      val measurements = measurer.measure[T](
         ctx, ctx(Key.exec.benchRuns),
         setup, teardown, regen, b
       )
   
-      resultFunction(measurements)
+      resultFunction(measurements.map(_.value))
```

Comment:
That is very weird - I'm not sure why this would happen.
And you are sure that you cleared the temporary folder before you did the change?